daemon/execdriver/lxc/driver.go

```diff
@@ -87,6 +87,12 @@ func (d *driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, startCallba
 		"-n", c.ID,
 		"-f", configPath,
 	}
+
+	// From lxc>=1.1 the default behavior is to daemonize containers after start
+	if d.version() >= "1.1" {
```

Comment:
If `"1.12.0" >= "1.2"` returns `false` then we can't rely on that as a check
On Feb 28, 2015 7:25 PM, "Vincent Batts" <vbatts@hashbangbash.com> wrote:

> Wat...
> On Feb 28, 2015 2:21 AM, "Tianon Gravi" <notifications@github.com> wrote:
>
>> In daemon/execdriver/lxc/driver.go
>> <https://github.com/docker/docker/pull/10527#discussion_r25555205>:
>>
>> > @@ -87,6 +87,12 @@ func (d *driver) Run(c *execdriver.Command, pipes *execdriver.Pipes, startCallba
>> >  		"-n", c.ID,
>> >  		"-f", configPath,
>> >  	}
>> > +
>> > +	// From lxc>=1.1 the default behavior is to daemonize containers after start
>> > +	if d.version() >= "1.1" {
>>
>> More like https://play.golang.org/p/Vjzpva-Hqj
>>
>> false, 1.1.1 >= 1.2
>> true, 1.2.3 >= 1.2
>> false, 1.12.0 >= 1.2
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/docker/docker/pull/10527/files#r25555205>.
>>
>